R/utilities-figures.R

```diff
@@ -342,8 +342,9 @@ createPlotsFromExcel <- function(
   plotGrids <- apply(dfPlotGrids, 1, \(row) {
     plotGridConfiguration <- .createConfigurationFromRow(
       defaultConfiguration = defaultPlotGridConfig,
-      row[!(names(row) %in% c("name", "plotIDs"))]
+      row[!(names(row) %in% c("name", "plotIDs", "title"))]
```

Comment:
`title` column was processed as other columns, where we follow the logic that "list" of values are separated by a comma. In case of a title (and file name in the next changed code chunk), there is always only one value and should not be split.